lib/dead_code.rb

```diff
@@ -0,0 +1,13 @@
+module DeadCode
+  NotDeadCodeError = Class.new(StandardError)
+
+  def dead_code!
+    error = NotDeadCodeError.new("NOT dead code: #{caller.first.inspect}")
```

Comment:
I'm probably being dense, but what does the exception message say "NOT dead code"?

Reply:
Nevermind, I get it now.